xmtp/migrations/2023-06-30-063813_create_users_and_conversations/up.sql

```diff
@@ -1,8 +1,9 @@
 CREATE TABLE users (
-    user_address TEXT PRIMARY KEY NOT NULL,
-    created_at BIGINT NOT NULL,
-    last_refreshed BIGINT NOT NULL
+  user_address TEXT PRIMARY KEY NOT NULL,
```

Comment:
are we re-using some of the tables from XMTP?

Reply:
This schema below is all we have a need for at the moment. Because OpenMLS has its own database structure with the KeyValue store, some of the tables from the old `xmtp` schema simply aren't necessary.